.vscode/targets.log

```diff
@@ -0,0 +1,366 @@
+make all --print-data-base --no-builtin-variables --no-builtin-rules --question
```

Comment:
Same as above.

Can we remove the .vscode files which got uploaded
Please add the following entry in .gitignore -> `.vscode`.